tests/CborTest.cpp

```diff
@@ -184,8 +184,8 @@ static int s_decode_timestamp_helper(Cbor::CborDecoder &decoder, std::chrono::sy
         {
             double double_val = decoder.PopNextFloatVal().value();
             std::chrono::duration<double, std::chrono::seconds::period> timestamp(double_val);
-            outTimePoint =
-                std::chrono::system_clock::time_point(std::chrono::duration_cast<std::chrono::milliseconds>(timestamp));
+            outTimePoint = std::chrono::system_clock::time_point(
+                std::chrono::duration_cast<std::chrono::system_clock::duration>(timestamp));
```

Comment:
we were casting from float->int->float here, now it's float->float->float

well ... some of those "float" are weird "std chrono ratio" things, but you know what I mean